### PR TITLE
 GPU Mining Apps 2.0

### PR DESCRIPTION
**Pull request by 88plug**
Add the most popular GPU mining apps:

- [XMRig GPU](https://xmrig.com/)
- [NBMiner](https://nbminer.com/)
- [Nanominer](https://nanominer.org/)
- [MoneroOcean GPU with XMR payout](https://github.com/MoneroOcean/xmrig/releases)
- [Bminer](https://bminer.me/examples/)
- [CryptoDredge](https://github.com/CryptoDredge/miner)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -236,14 +236,32 @@ Awesome DeFi apps you can deploy on Akash
...
 - [PacketStream](packetstream)
 - [Peer2Profit](peer2profit)
 - [pkt.cash](pkt-miner)
 - [RainbowMiner](rainbowminer)
 - [Raptoreum](raptoreum-miner)
 - [Traffmonetizer](traffmonetizer)
-- [xmrig](xmrig)
+- [XMRig CPU](xmrig)
+
+### Mining - GPU
```
Comment (andy108369) [inline README.md:248]: We already have this section.

**Review by andy108369 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (88plug) [reply README.md:248]: In search in Cloudmos it's a problem, you can't distinguish. It's important for seo as well. Please do not block because of this.

Comment (88plug) [reply README.md:248]: Actually I need to add to RainbowMiner - should be there as well.

Comment (arno01) [reply README.md:248]: image

**Review by arno01 — APPROVED** [review]

**Review by arno01 — APPROVED** [review]

**Review by andy108369 — APPROVED** [review]

**Pull request merged**